utils/build/webpack.common.js

```diff
@@ -81,9 +84,29 @@ const withStylingModuleLoader = returnModuleRuleWithConfig(
   ]
 );
 
+const withTSModuleLoader = (tsConfigFile) =>
+  returnModuleRuleWithConfig(
+    {
+      test: /\.(tsx|ts)?$/,
```

Comment:
Can change this to `/\.(js|tsx|ts)?$/` and then we can remove babel-loader and withJSModuleLoader